src/main/java/net/mcreator/ui/modgui/KeyBindGUI.java

```diff
@@ -83,8 +83,9 @@ public KeyBindGUI(MCreator mcreator, ModElement modElement, boolean editingMode)
 				L10N.label("elementgui.keybind.key_binding_name")));
 		enderpanel.add(keyBindingName);
 
-		enderpanel.add(HelpUtils.wrapWithHelpButton(this.withEntry("keybinding/category"),
-				L10N.label("elementgui.keybind.key_binding_category")));
+		enderpanel.add(HelpUtils.wrapWithHelpButton(this.withEntry("keybinding/category"), L10N.label("elementgui.keybind.key_binding_category"),
+				L10N.t("tab.workspace"), L10N.t("workspace.category.localization"),
+				L10N.t("workspace.localization.add_entry"), keyBindingCategoryKey.getSelectedItem()));

```

Comment:
keyBindingCategoryKey.getSelectedItem() will not update when category is changed. Consider not using this system in places like this as this can be confusing. Applies to some other cases too.

Reply:
I already know the solution: I just need to somehow pass text pane for the help context into the `wrapWithHelpButton` as a parameter so I can use an `ActionListener` to update such parameters.

Reply:
That would be a bit messy, consider using suppliers in this case instead.

Reply:
I have idea how to implement this.